test/load_torchscript_model.py

```diff
@@ -0,0 +1,5 @@
+import sys
+import torch
+
+if __name__ == '__main__':
+    print(torch.jit.load(sys.argv[1]))
```

Comment:
Is there a way to distinguish failures in BC/FC scenario since they are going to have same stacktrace and no additional information attached for developers to debug?

Reply:
Good point. Done by adding messages in `test.sh`